src/cfml/system/config/server.schema.json

```diff
@@ -389,6 +535,166 @@
 							"default": {}
 						}
 					}
+				},
+				"fileCache": {
```

Comment:
FWIW, this feature was never actually released due to bugs in Undertow that remain unfixed :/

Reply:
That explains why I couldn't find any documentation on it. I'll remove it for now.